gc/structs/ForwardedHeader.hpp

```diff
@@ -40,7 +42,10 @@
    This hint is not necessary for correctness of copying protocol, it's just an optimization to avoid visiting destination object header
    in cases when it's likely not in data cash (GC thread encountering already forwarded object) */
 #define OMR_BEING_COPIED_HINT 2
-#define OMR_SELF_FORWARDED_TAG J9_GC_MULTI_SLOT_HOLE
+/* We define OMR_SELF_FORWARDED_TAG as 1 and not as J9_GC_MULTI_SLOT_HOLE (due to DDR pre-preprocessor limitation of not processing #includes properly) */
```

Comment:
Be more descriptive for future readers:

OMR_SELF_FORWARDED_TAG overloads existing J9_GC_MULTI_SLOT_HOLE 0x1. In a rare case, we may need iterating a Evacuate area, that may have all combinations of: non-forwarded objects (0x4 not set, 0x1 not set), strictly forwarded objects (0x4 set, 0x1 not set), self forwarded objects (0x4 set, 0x1 set) and free memory chunks (0x4 not set and 0x1 set). GC_ObjectHeapIteratorAddressOrderedList  is aware of this overloading and will properly interpret when 0x1 is set.

However, we define OMR_SELF_FORWARDED_TAG directly as 1 and not as J9_GC_MULTI_SLOT_HOLE (due to DDR pre-preprocessor limitation of not processing #includes properly)